lib/ext/provider.rb

```diff
@@ -19,6 +19,7 @@ def self.generate_provider(options = {})
     prov.family_name = NAMES_RANDOM['last'].sample
     prov.npi = NpiGenerator.generate
     prov.tin = rand.to_s[2..10]
+    prov.ccn = rand(1..66).to_s.ljust(2, '0') + rand(1..9899).to_s
```

Comment:
It is actually different, but because I screwed up. I should have used `rjust` here. Thanks for the catch.